Sprint-2/stretch/count-words.js

```diff
@@ -26,3 +26,16 @@
 
 3. Order the results to find out which word is the most common in the input
 */
+
+function countWords(str){
```

Comment:
This code works with all concepts as expected. There is a possibility of improving the code. Please check the need of any conditional statements and it's impact

Reply:
I have updated  it